### PR TITLE
Use same tooltip for Graph and Tree views for TaskInstances

### PR DESCRIPTION
**Pull request by ashb**
They had _similar_ info, but they weren't identical and should have
been.

To not include the 250kb of moment-timezone in more than one Webpack
output bundle I changed how that is imported/handled. There may be a
better way of doing this in webpack, but I couldn't find it, and this
isn't horrible.

#### Graph TI:

**Before:**
before_graph_ti

**After:**
after_graph_ti

(i.e. unchanged)

#### Tree TI:
**Before:**
before_tree_ti

**After:**
after_tree_ti

#### Tree DagRun

**Before**: BROKEN ON MASTER -- no tooltip was displayed! Oops.

**After:**
after_tree_dagrun

---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by ashb** [comment]
Whops, just notcied the `Local: UTC` that should just be `UTC`

File: airflow/www/static/js/datetime-utils.js [hunk 1]
```diff
@@ -16,41 +16,15 @@
...
-
-  // Generate DAG's Start and End Date
-  if (dagTZ !== 'UTC' && dagTZ !== localTZ) {
-    tooltipHTML += `<br><strong>DAG's TZ: ${moment.tz(dagTZ).format(tzFormat)}</strong><br>`;
-    tooltipHTML += makeDateTimeHTML(startDate.tz(dagTZ), endDate.tz(dagTZ));
-  }
-
-  return tooltipHTML
-};
-
```
Comment (ashb) [inline airflow/www/static/js/datetime-utils.js:53]: I moved these fns in to a new "task-instances.js" as they are specific to there, and not general "datetime" utils.

File: airflow/www/static/js/graph.js [hunk 2]
```diff
@@ -16,8 +16,8 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { generateTooltipDateTime, converAndFormatUTC, secondsToString } from './datetime-utils';
-import { escapeHtml } from './base';
+// global tiTooltip, taskTip
```
Comment (ashb) [inline airflow/www/static/js/graph.js:19]: Global as it's included in another file that is sourced.

If I used a normal import webpack would include it in two files.

**Review by kaxil — APPROVED** [review]

File: airflow/www/static/js/task-instances.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+export default function tiTooltip(ti) {
+  let tt = '';
+  if(ti.task_id !== undefined) {
+    tt += `Task_id: ${escapeHtml(ti.task_id)}<br>`;
+  }
+  tt += `Run: ${formatDateTime(ti.execution_date)}<br>`;
+  if(ti.run_id !== undefined) {
+    tt += `run_id: <nobr>${escapeHtml(ti.run_id)}</nobr><br>`;
```
Comment (BasPH) [inline airflow/www/static/js/task-instances.js]: For consistency sake, let's start with capital.
```suggestion
    tt += `Run_id: <nobr>${escapeHtml(ti.run_id)}</nobr><br>`;
```

**Review by BasPH — DISMISSED** [review]

File: airflow/www/static/js/task-instances.js (hunk 3 above)
Comment (ashb) [reply airflow/www/static/js/task-instances.js]: I'll make it `Run Id` to match how we show the column in Browse -> Dag Runs.

**Pull request merged**